ubuntu/debian/libgz-utils-log.install

```diff
@@ -0,0 +1 @@
+usr/lib/*/lib*-utils[0-99]-log.so.*
```

Comment:
I did a test run and it looks like there are some warnings: https://build.osrfoundation.org/job/gz-utils3-debbuilder/513/console

```
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3 libgz-utils3-log usr/lib/x86_64-linux-gnu/libgz-utils3-log.so.3.0.0
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3-dev libgz-utils3-log-dev usr/include/gz/utils3/gz/utils/log.hh
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3-dev libgz-utils3-log-dev usr/include/gz/utils3/gz/utils/log/Export.hh
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3-dev libgz-utils3-log-dev usr/include/gz/utils3/gz/utils/log/Logger.hh
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3-dev libgz-utils3-log-dev usr/include/gz/utils3/gz/utils/log/SplitSink.hh
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3-dev libgz-utils3-log-dev usr/include/gz/utils3/gz/utils/log/detail/Export.hh
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3-dev libgz-utils3-log-dev usr/lib/x86_64-linux-gnu/cmake/gz-utils3-log/gz-utils3-log-config-version.cmake
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3-dev libgz-utils3-log-dev usr/lib/x86_64-linux-gnu/cmake/gz-utils3-log/gz-utils3-log-config.cmake
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3-dev libgz-utils3-log-dev usr/lib/x86_64-linux-gnu/cmake/gz-utils3-log/gz-utils3-log-targets-relwithdebinfo.cmake
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3-dev libgz-utils3-log-dev usr/lib/x86_64-linux-gnu/cmake/gz-utils3-log/gz-utils3-log-targets.cmake
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3-dev libgz-utils3-log-dev usr/lib/x86_64-linux-gnu/pkgconfig/gz-utils3-log.pc
```

Reply:
ok made a few changes. The build is green now: https://build.osrfoundation.org/job/gz-utils3-debbuilder/517/